public/index.html

```diff
@@ -0,0 +1,741 @@
+        </section>
+        <nav class="home-nav">
+          <ul>
+            <li><a class="home-nav-a" href="#home">HOME</a></li>
+            <li><a class="home-nav-a" href="#about">ABOUT</a></li>
+            <li><a class="home-nav-a" href="#features">FEATURES</a></li>
+            <li><a class="home-nav-a" href="#reviews">REVIEWS</a></li>
+            <li><a class="home-nav-a" href="#screens">SCREENS</a></li>
+            <li><a class="home-nav-a" href="#blog">BLOG</a></li>
+            <li><a class="home-nav-a" href="#support">SUPPORT</a></li>
+            <li class="icon-seek"></li>
+          </ul>
+        </nav>
+      </section>
+      <section class="a2-home-top">
+        <section class="home-h1-section">
+          <h1 class="home-h1 a2-h1">Best Way To Present Your<br> Beautiful Mobile App</h1>
+          <p class="home-p home-p-first-sentance">AppBox is full of awesome features. It's designed for</p>
+          <p class="home-p home-p-second-sentance"><strong>App Landing and App Store site</strong></p>
+          <section class="home-video">
+            <div class="icon-play home-video-border"></div>
+            <div><a class="home-intro-vid" href="">Watch the Intro Video</a></div>
+          </section>
+        </section>
+        <section class="home-phone-pic">
+          <img src="./img/overlayed-phones.png" class="blue-phone" alt="a blue iPhone on top of a green one">
+        </section>
+      </section>
+      <section class="a3-home-middle">
+        <svg class="slant" xmlns="http://www.w3.org/2000/svg" version="1.1" width="100%" height="100" viewBox="0 0 100 102" preserveAspectRatio="none">
+          <path d="M0 0 L0 100 L100 0 Z" />
+        </svg>
+        <h1 class="home-h1">GET IT FOR FREE</h1>
+        <p class="home-middle-p">Choose your native platform and download the app FREE!</p>
+        <ul class="home-google-ios">
+          <li><a class="submit-button" href="https://play.google.com/store?hl=en"><span class="icon-Googleplay"></span>  Google Play</a></li>
+          <li><a class="submit-button" href="http://www.apple.com/itunes/charts/free-apps/"><span class="icon-iOS"></span>  iOS App Store</a></li>
+        </ul>
+      </section>
+      <section class="a4-home-bottom">
+      <svg class="slant" xmlns="http://www.w3.org/2000/svg" version="1.1" width="100%" height="100" viewBox="0 0 100 102" preserveAspectRatio="none">
+        <path d="M0 0 L0 100 L100 0 Z" />
+      </svg>
+      <div class="underlined-heading">
+        <h1 class="underlined-heading-text">Some Amazing Features</h1>
+        <div class="underlined-heading-underline"></div>
+      </div>
+      <section class="home-feature">
+      <ul class="home-feature-list">
+        <li>
+        <figure class="home-feature home-feature-laptop">
+          <div class="container-icon">
+            <p class="icon-laptop"></p>
+          </div>
+          <h5 class="home-feature-h5">RESPONSIVE DESIGN</h5>
```

Comment:
Also, because I really want to drive this home, your classnames should _not_ include a direct reference to the element on which they're placed (in this case, the `-h5`). Here I would go with `home-feature-heading`.